examples/connext_dds/get_publishers/c++/Foo_publisher.cxx

```diff
@@ -86,6 +86,7 @@ extern "C" int publisher_main(int domainId, int sample_count)
 {
     DDSDomainParticipant *participant = NULL;
     DDSPublisher *publisher = NULL, *publisher2 = NULL;
+    int count = 0;
```

Comment:
There is a comment for this same line in https://github.com/rticommunity/rticonnextdds-examples/pull/356/files pr